cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -54,667 +46,308 @@
+        f"{report_value} does not match expected value {estimated_value}.")
+
+
+def test_validate_memory_alloc_cost(chargeback_costs, chargeback_report):
+    """Test to validate cost for memory allocation
+
+    Polarion:
+        assignee: tpapaioa
+        initialEstimate: 1/4h
+        casecomponent: CandU
+    """
+    report_value = get_report_value(chargeback_report, 'Memory Allocated Cost')
+    estimated_value = chargeback_costs['memory_alloc_cost']
+    assert compare_values(estimated_value, report_value), ("Reported memory allocation cost "
+        f"{report_value} does not match expected value {estimated_value}.")
 
 
+def test_validate_storage_alloc_cost(chargeback_costs, chargeback_report):
+    """Test to validate cost for storage allocation
+
+    Polarion:
+        assignee: tpapaioa
+        initialEstimate: 1/4h
+        casecomponent: CandU
+    """
+    report_value = get_report_value(chargeback_report, 'Storage Allocated Cost')
+    estimated_value = chargeback_costs['storage_alloc_cost']
+    assert compare_values(estimated_value, report_value), ("Reported storage allocation cost "
+        f"{report_value} does not match expected value {estimated_value}.")
+
+
+@pytest.mark.provider(gen_func=providers,
+                      filters=[cloud_and_infra, not_cloud],
+                      scope='module')
+def test_validate_cpu_usage(resource_usage, chargeback_report):
+    """Test to validate CPU usage. This metric is not collected for cloud providers.
+
     Polarion:
         assignee: tpapaioa
         caseimportance: medium
         casecomponent: Reporting
-        initialEstimate: 1/12h
+        initialEstimate: 1/4h
     """
-    for groups in chargeback_report_custom:
-        if groups["CPU Used Cost"]:
-            est_cpu_cost = chargeback_costs_custom['cpu_used_cost']
-            report_cost = groups["CPU Used Cost"]
-            assert cost_comparison(est_cpu_cost, report_cost), 'CPU report cost does not match'
-            break
+    report_value = get_report_value(chargeback_report, 'CPU Used')
+    estimated_value = resource_usage['cpu_used']
+    if 'GHz' in report_value:
+        estimated_value = estimated_value * math.pow(10, -3)
+    report_value = report_value.replace('MHz', '').replace('GHz', '')
+    assert compare_values(estimated_value, report_value), ("Reported CPU usage "
+        f"{report_value} does not match expected value {estimated_value}.")
 
 
 @pytest.mark.rhv1
 @pytest.mark.provider(gen_func=providers,
                       filters=[cloud_and_infra, not_scvmm, not_ec2_gce],
                       scope='module')
-def test_validate_custom_rate_memory_usage_cost(chargeback_costs_custom, chargeback_report_custom):
-    """Test to validate memory usage cost.
-       Calculation is based on custom Chargeback rate.
+def test_validate_memory_usage(resource_usage, chargeback_report):
+    """Test to validate memory usage. This metric is not collected for GCE, EC2.
+
```

Comment:
This metric is now collected for EC2. So, this test should be run for EC2 as well.